### ISSUE TITLE
#TheDress and Why Web Accessibility Matters to Web Developers

### ISSUE THREAD

**Issue by melodyma** [cascadiajs/2015.cascadiajs.com#186]
## #TheDress and Why Web Accessibility Matters to Web Developers

* Name      : Melody Ma
* Location  : Vancouver,BC
* Email     : melody.yy.ma@gmail.com
* Twitter   : [@melodyma](https://twitter.com/melodyma)
* GitHub    : [melodyma](https://github.com/melodyma)
* Url(s)    :  [medium.com/@melodyma](https://medium.com/@melodyma)

## The story you'd like to tell
Are you #teamwhite&gold or #teamblue&black? Regardless, this ridiculous internet meme tore the internet apart and shows just why as web developers, we need to build web products that are accessible for everyone regardless of their physical and cognitive abilities. This talk will give an overview of what web accessibility means, why it matters from a technical, legal, ux and business perspective, and how your organization can start integrating #a11y in your processes and projects. We'll look at case studies of organizations like Target, Tesco and the Government of Canada where they failed in providing adequate web accessibility, paid dearly for it, and the outcomes of each case.

## Speaker Bio

![](https://en.gravatar.com/userimage/77985841/e77dc268e32b4e52b932bd68b872d8dd.jpg?size=200)

Melody is a web developer and previously a software product manager. She is a member of the W3C Web Accessibility Initiative Education and Outreach Work Group and is a contributor to the Web Experience Toolkit (WET) open source project led by the Government of Canada, which is an award-winning framework for building accessible websites.
